malariagen_data/plasmodium.py

```diff
@@ -188,3 +191,34 @@ def variant_calls(self, extended=False, inline_array=True, chunks="native"):
         ds = xarray.Dataset(data_vars=data_vars, coords=coords)
 
         return ds
+
+    def genome_features(self, attributes=("ID", "Parent", "Name", "alias")):
+        """Access genome feature annotations.
+
+        Parameters
+        ----------
+        attributes : list of str, optional
+            Attribute keys to unpack into columns. Provide "*" to unpack all attributes.
+
+        Returns
+        -------
+        df : pandas.DataFrame
+
+        """
+        # Attributes
+        if type(attributes) not in [tuple, list] and attributes != "*":
+            raise TypeError("'attributes' must be a list, tuple, or '*'")
+        if attributes is not None:
+            attributes = tuple(attributes)
```

Comment:
Could you factor this out into a separate function so we can use it in Ag3 and Amin1. E.g., in malariagen_data.util:

```python
def _prep_geneset_attributes_arg(attributes):
    if type(attributes) not in [tuple, list] and attributes != "*":
        raise TypeError("'attributes' must be a list, tuple, or '*'")
    if attributes is not None:
        attributes = tuple(attributes)
    return attributes
```

Then code here (and in Ag3 and Amin1) could become:

```suggestion
        attributes = _prep_geneset_attributes_arg(attributes)
```

Reply:
I've moved this out into util.py now